packages/manager/apps/web/client/app/hosting/ftp/FTP.html

```diff
@@ -298,66 +298,54 @@
                         </dd>
 
                         <dt
-                            data-translate="hosting_tab_FTP_dashboard_server_link"
+                            data-translate="hosting_tab_FTP_SSH_dashboard_server"
                         ></dt>
                         <dd class="mb-3">
-                            <a
-                                data-ng-href="ftp://{{ftp.url}}:{{ftp.port}}/"
-                                target="_blank"
-                                title="{{ 'hosting_dashboard_service_home_ftp_newtab' | translate }}"
-                            >
-                                <span data-ng-bind="ftpUrl"></span>
-                                <span
-                                    class="fa fa-external-link"
-                                    aria-hidden="true"
-                                ></span>
-                            </a>
+                            <oui-clipboard
+                                class="w-75"
+                                data-model="ssh.url"
+                            ></oui-clipboard>
                         </dd>
 
-                        <dt
-                            data-translate="hosting_tab_FTP_dashboard_clusterip"
-                        ></dt>
-                        <dd
-                            class="mb-3"
-                            data-ng-bind="ctrlHostingTabFTP.ftpInformations.hostingIp"
-                        ></dd>
-
-                        <dt
-                            data-translate="hosting_dashboard_service_home_ssh"
-                        ></dt>
+                        <dt data-translate="hosting_tab_FTP_ports"></dt>
                         <dd class="mb-3">
-                            <a
-                                data-ng-href="ssh://{{ssh.url}}:{{ssh.port}}/"
-                                target="_blank"
-                                title="{{ 'hosting_dashboard_service_home_ssh_newtab' | translate }}"
-                            >
-                                <span data-ng-bind="sshUrl"></span>
+                            <div>
+                                <span
+                                    class="col-md-2"
```

Comment:
**issue:** Columns (`.col-*-*`) can only be children of `.rows`.

see: https://github.com/twbs/bootlint/wiki/E014

Why not prefer using `translate-values` directive here?

As an example:
```html
<span
    class="d-block"
    data-translate="hosting_tab_FTP_FTP_port"
    data-translate-values="{ port : ftp.port }"
></span>
```

```json
{
  "hosting_tab_FTP_FTP_port": "FTP : {{ port }}"
}
```

see: https://angular-translate.github.io/docs/#/guide/06_variable-replacement#variable-replacement_variable-replacement-in-translate-directive

Reply:
Hello,

thanks for the review. Ok for columns children of row.
I didnt know about translate-values directive. But using it, I find the display less clear for the user (alignment is lost). Is it ok to keep col and row ?

<img width="345" alt="image" src="https://user-images.githubusercontent.com/89460513/142837785-7cb981e2-b441-49cf-911d-a4baa70809d0.png">

<img width="345" alt="image" src="https://user-images.githubusercontent.com/89460513/142838126-d3b4040d-5de0-4d2c-a8c8-40f185ccbdd6.png">

Reply:
Maybe this https://ovh.github.io/ovh-ui-kit/?path=/docs/design-system-content-descriptions--horizontal could suit your needs

Reply:
Hello,

thanks for the tip, I keep in mind horizontal description, it can be useful. However, for this pull request, I find the display of nested descriptions not so clear for user. What do you think ?

<img width="491" alt="image" src="https://user-images.githubusercontent.com/89460513/143256207-b889f227-4379-4c86-9edb-023b06392444.png">

Reply:
@lucas-chaigne maybe you could play with the list width, to avoid having the description values to be set at the farest position possible in the container?

Reply:
Hello,

@JeremyDec the display with the current code of the pull request is good to me (see pic below). Is it a bad practice to use col and row to obtain it ? 
I fixed issue "Columns (.col-*-*) can only be children of .rows." found by Antoine so the syntax should be ok now.

current display with row and col : 
![image](https://user-images.githubusercontent.com/89460513/143416549-758cff63-64d1-4db3-9b33-ebcc5541cc82.png)

Reply:
pull request is updated using horizontal description and list width

<img width="262" alt="image" src="https://user-images.githubusercontent.com/89460513/143613596-faef60e7-21dc-4fa4-8461-b37d9acb1db0.png">